### PR TITLE
Add line break functionality to instruction and description

### PR DESCRIPTION
**Pull request by iwootten**
### What is the context of this PR?

Updated the questionnaires to use a list for the description. https://trello.com/c/On5OBCNm/3939-add-line-break-functionality-in-question-description-m

### How to review

Ensure the schemas build and can be validated using the appropriate validator branch.

### Checklist

- [x] Jsonnet files conform to the latest [style guide](/ONSdigital/eq-questionnaire-schemas/blob/master/style_guide.md)

#### England

- [Household](https://test-launcher.gcp.dev.eq.ons.digital/quick-launch?language_code=en&survey=CENSUS&form_type=H&region_code=GB-ENG&url=https://storage.googleapis.com/eq-questionnaire-schemas-artifacts/make-description-list/schemas/en/census_household_gb_eng.json)

- [Individual](https://test-launcher.gcp.dev.eq.ons.digital/quick-launch?language_code=en&survey=CENSUS&form_type=I&region_code=GB-ENG&url=https://storage.googleapis.com/eq-questionnaire-schemas-artifacts/make-description-list/schemas/en/census_individual_gb_eng.json)

- [CCS](https://test-launcher.gcp.dev.eq.ons.digital/quick-launch?language_code=en&survey=CENSUS&form_type=I&region_code=GB-ENG&url=https://storage.googleapis.com/eq-questionnaire-schemas-artifacts/make-description-list/schemas/en/ccs_household_gb_eng.json)

#### Northern Ireland

- [Household](https://test-launcher.gcp.dev.eq.ons.digital/quick-launch?language_code=en&survey=CENSUS&form_type=I&region_code=GB-NIR&url=https://storage.googleapis.com/eq-questionnaire-schemas-artifacts/make-description-list/schemas/en/census_household_gb_nir.json)

- [Individual](https://test-launcher.gcp.dev.eq.ons.digital/quick-launch?language_code=en&survey=CENSUS&form_type=I&region_code=GB-NIR&url=https://storage.googleapis.com/eq-questionnaire-schemas-artifacts/make-description-list/schemas/en/census_individual_gb_nir.json)

### CODE REVIEW

**Comment by iwootten** [comment]
[Add line break functionality in question description (M)](https://trello.com/c/On5OBCNm/3939-add-line-break-functionality-in-question-description-m)

File: source/jsonnet/england-wales/ccs/blocks/who-lives-here/anyone_else_list_collector.jsonnet [hunk 1]
```diff
@@ -67,7 +67,9 @@ local editQuestion(questionTitle) = {
         type: 'General',
         id: 'anyone-usually-live-at-question',
         title: questionTitle,
-        description: 'Remember to only include those people who share cooking facilities <strong>and</strong> share a living room, <strong>or</strong> sitting room, <strong>or</strong> dining area.<p><strong>Anyone else is not part of your household and will be interviewed separately.</strong></p>',
+        description: [
+          'Remember to only include those people who share cooking facilities <strong>and</strong> share a living room, <strong>or</strong> sitting room, <strong>or</strong> dining area.<p><strong>Anyone else is not part of your household and will be interviewed separately.</strong></p>',
```
Comment (ajmaddaford) [inline source/jsonnet/england-wales/ccs/blocks/who-lives-here/anyone_else_list_collector.jsonnet:71]: This should be two strings (with the `p` tags removed).

File: source/jsonnet/england-wales/ccs/blocks/who-lives-here/anyone_else_list_collector.jsonnet [hunk 2]
```diff
@@ -127,7 +129,9 @@ local editQuestion(questionTitle) = {
             placeholders.censusDate,
           ],
         },
-        description: 'Remember to only include those people who share cooking facilities <strong>and</strong> share a living room, <strong>or</strong> sitting room, <strong>or</strong> dining area.<p><strong>Anyone else is not part of your household and will be interviewed separately.</strong></p>',
+        description: [
+          'Remember to only include those people who share cooking facilities <strong>and</strong> share a living room, <strong>or</strong> sitting room, <strong>or</strong> dining area.<p><strong>Anyone else is not part of your household and will be interviewed separately.</strong></p>',
```
Comment (ajmaddaford) [inline source/jsonnet/england-wales/ccs/blocks/who-lives-here/anyone_else_list_collector.jsonnet:133]: This should be two strings (with the `p` tags removed).

**Comment by ajmaddaford** [comment]
Given the issues this will cause as a breaking change, we should consider also changing `instruction` to a list at the same time.

File: Pipfile [hunk 3]
```diff
@@ -7,7 +7,7 @@ verify_ssl = true
 coloredlogs = "*"
 
 [packages]
-eq-translations = {editable = true,git = "https://github.com/ONSDigital/eq-translations.git",ref = "v3.1.0"}
+eq-translations = {editable = true,git = "https://github.com/ONSDigital/eq-translations.git",ref = "make-description-list"}
```
Comment (JamesGardiner) [inline Pipfile:10]: reminder to revert before merge

File: scripts/run_validator.sh [hunk 4]
```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env bash
 
-tag=latest
+tag=make-description-list
```
Comment (JamesGardiner) [inline scripts/run_validator.sh:3]: reminder to revert before merge

File: source/jsonnet/england-wales/ccs/blocks/who-lives-here/who_to_interview_note.jsonnet [hunk 5]
```diff
@@ -3,7 +3,7 @@
   id: 'who-to-interview-note',
   content: {
     title: '<em>Interviewer Note</em>',
-    instruction: 'If the respondent was not living at the property on census night, but other current household members were, you must speak to one of those household members instead. <p>If none of those household members are available, you must return to the address to interview one of them at a later date.</p>',
+    instruction: ['If the respondent was not living at the property on census night, but other current household members were, you must speak to one of those household members instead. <p>If none of those household members are available, you must return to the address to interview one of them at a later date.</p>']
```
Comment (ajmaddaford) [inline source/jsonnet/england-wales/ccs/blocks/who-lives-here/who_to_interview_note.jsonnet]: This needs to be split into two separate lines and the `p` tags removed.

Comment (iwootten) [reply source/jsonnet/england-wales/ccs/blocks/who-lives-here/who_to_interview_note.jsonnet]: Think you've commented on an old commit here.

**Pull request merged**